gwpy/frequencyseries/frequencyseries.py

```diff
@@ -279,27 +279,23 @@ def interpolate(self, df, left=0, right=0):
             desired frequency resolution of the interpolated `FrequencySeries`,
             in Hz
 
-        left : `float`
-            value to return for f < flow, default: 0
-
-        right : `float`
-            value to return for f > fhigh, default: 0
-
         Returns
         -------
         out : `FrequencySeries`
             the interpolated version of the input `FrequencySeries`
 
         See Also
         --------
-        :func:`numpy.interp`
+        :func:`~numpy.interp`
```

Comment:
Can you change this to be:

```
        See Also
        ---------
        numpy.interp
```
(i.e. remove the ticks and tildes, please)?

Reply:
@duncanmmacleod, this is done, I'm actually not sure why I put those there in the first place 😱